src/types/module.ts

```diff
@@ -65,6 +65,39 @@ export interface UploadcareOptions {
   [key: string]: any
 }
 
+export interface IPXModifiers extends ImageModifiers {
```

Comment:
Would you please add a comment above to ipx docs and if had time open an issue upstream to remember adding operation types from upstream? 🙏